llvm/lib/Target/AMDGPU/SIFoldOperands.cpp

```diff
@@ -219,10 +219,8 @@ bool SIFoldOperands::canUseImmWithOpSel(FoldCandidate &Fold) const {
   default:
     return false;
   case AMDGPU::OPERAND_REG_IMM_V2FP16:
-  case AMDGPU::OPERAND_REG_IMM_V2BF16:
```

Comment:
Rebase issue? This was added in #82435.

Reply:
Thanks for catching! not sure if I ever touched that but it seems concerning if rebase did that.